README.md

```diff
@@ -104,14 +92,16 @@ make install
 to install executables and dependencies into the default install directory (**`~/chronolog`**).
 
 ## Configuration files
-- **Default Configuration:** ChronoLog executables require a configuration file (`default_conf.json.in`). Modify this template according to your preferences.
-- **Installation Process:** The default installation copies and renames `default_conf.json.in` to `conf` in the install directory.
-- **Using Configuration:** Pass the configuration file to executables with `-config default_conf.json`.
-
 
+- **Default Configuration:** ChronoLog executables require a configuration file. Modify this template (`default_conf.json.in`) according to your preferences.
+- **Installation Process:** The installation copies and renames default_conf.json.in to conf/default_conf.json in the
```

Comment:
pls put default_conf.json.in and conf/default_conf.json inside pairs of `` to highlight them.